static/examples/hashing-tool/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Hashing Tool</title>
+    <link rel="stylesheet" type="text/css" href="./app.css" />
+    <link rel="shortcut icon" href="/img/favicon.ico" />
+    <!-- <script src="https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js"></script> -->
+    <script src="./uid2-sdk-3.3.0.js"></script>
+    <!-- <script type="module">
+      //import * as exportsTest from './uid2-sdk-3.3.0.js';
+      //console.log(exportsTest);
+      //import { hashAndEncodeIdentifier } from '@uid2/uid2-sdk';
+       //import { hashAndEncodeIdentifier } from './uid2-sdk-3.3.0.js'
+      
+    </script> -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  </head>
+  <script>
+    console.log('Initializing example.');
+    
+
+    
+
+    async function updateGuiElements(input) {
+      console.log('Updating displayed values.');
+      const input_type = document.querySelector('input[name="toggle_input_type"]:checked').value;
```

Comment:
you're using camelCase for other variable names.  Maybe change this one to make it consistent.